Doc/howto/logging-cookbook.rst

```diff
@@ -765,13 +765,70 @@ serialization.
 Running a logging socket listener in production
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To run a logging listener in production, you may need to use a process-management tool
-such as `Supervisor <http://supervisord.org/>`_. `Here
-<https://gist.github.com/vsajip/4b227eeec43817465ca835ca66f75e2b>`_ is a Gist which
-provides the bare-bones files to run the above functionality using Supervisor: you
-will need to change the ``/path/to/`` parts in the Gist to reflect the actual paths you
-want to use.
-
+To run a logging listener in production, you may need to use a
```

Comment:
```suggestion
.. _socket-listener-gist: https://gist.github.com/vsajip/4b227eeec43817465ca835ca66f75e2b

To run a logging listener in production, you may need to use a
```

Single-source the long URL instead of embedding it two different places in the text, so its easier to update and harder to get out of sync. You could also put it after the first paragraph, or at the end of the section if you prefer; I just put it here for now where seemed easiest to find.